pinot-query-planner/src/main/java/org/apache/pinot/query/planner/serde/ProtoSerializationUtils.java

```diff
@@ -141,6 +147,10 @@ private static Plan.MemberVariableField serializeMemberVariable(Object fieldObje
       builder.setLiteralField(doubleField((Double) fieldObject));
     } else if (fieldObject instanceof String) {
       builder.setLiteralField(stringField((String) fieldObject));
+    } else if (fieldObject instanceof byte[]) {
+      builder.setLiteralField(bytesField(ByteString.copyFrom((byte[]) fieldObject)));
+    } else if (fieldObject instanceof GregorianCalendar) {
+      builder.setLiteralField(longField(((GregorianCalendar) fieldObject).getTimeInMillis()));
```

Comment:
should not use longField b/c we cannot differentiate whether it is a long or a Calendar object when DeSer